DistributionPackages/Neos.Presentation/Resources/Private/Fusion/Module/List/List.With.Preview.fusion

```diff
@@ -0,0 +1,136 @@
+                        label = 'E-commerce'
+                        color = 'green'
+                    }
+                }
+                20 = Neos.Fusion:DataStructure {
+                    image = Neos.Fusion:DataStructure {
+                        imageSource = Sitegeist.Kaleidoscope:DummyImageSource {
+                            width = 500
+                            height = 300
+                            text = '5:3'
+                        }
+                        altText = 'Lorem ipsum'
+                    }
+                    link = '#'
+                    label = 'Name of the client and his project'
+                    category = Neos.Fusion:DataStructure {
+                        label = 'E-commerce'
+                        color = 'green'
+                    }
+                }
+                30 = Neos.Fusion:DataStructure {
+                    image = Neos.Fusion:DataStructure {
+                        imageSource = Sitegeist.Kaleidoscope:DummyImageSource {
+                            width = 500
+                            height = 300
+                            text = '5:3'
+                        }
+                        altText = 'Lorem ipsum'
+                    }
+                    link = '#'
+                    label = 'Name of the client and his project'
+                    category = Neos.Fusion:DataStructure {
+                        label = 'Dev'
+                        color = '#b6b6b6'
+                    }
+                }
+                40 = Neos.Fusion:DataStructure {
+                    image = Neos.Fusion:DataStructure {
+                        imageSource = Sitegeist.Kaleidoscope:DummyImageSource {
+                            width = 500
+                            height = 300
+                            text = '5:3'
+                        }
+                        altText = 'Lorem ipsum'
+                    }
+                    link = '#'
+                    label = 'Name of the client and his project'
+                    category = Neos.Fusion:DataStructure {
+                        label = 'Content Platforms'
+                        color = 'blue'
+                    }
+                }
+                50 = Neos.Fusion:DataStructure {
+                    image = Neos.Fusion:DataStructure {
+                        imageSource = Sitegeist.Kaleidoscope:DummyImageSource {
+                            width = 500
+                            height = 300
+                            text = '5:3'
+                        }
+                        altText = 'Lorem ipsum'
+                    }
+                    link = '#'
+                    label = 'Name of the client and his project'
+                    category = Neos.Fusion:DataStructure {
+                        label = 'Content Platforms'
+                        color = 'blue'
+                    }
+                }
+            }
+        }
+    }
+
+    @propTypes {
```

Comment:
Please use [Fusion Validator Prototypes](https://github.com/PackageFactory/atomic-fusion-proptypes/tree/master#fusion-validator-prototypes)
Then you can also use `PropTypes:FromPrototype`